scripts/StateManager.js

```diff
@@ -67,6 +87,21 @@ function StateManager() {
 			}
 		});
 	}
+	
+	// Gets the current game based on the option, and weather if
+	// we're using a live weather option.
+	function getMusic() {
+		if(isLive()) {
+			if(weatherManager.getWeather() == "Rain")
+				return "new-leaf-raining";
```

Comment:
I'll get / upload them today. 
On Jan 30, 2016 4:27 AM, GammaGames <notifications@github.com> wrote:In scripts/StateManager.js:
> @@ -67,6 +87,21 @@ function StateManager() {
>  			}
>  		});
>  	}
> +	
> +	// Gets the current game based on the option, and weather if
> +	// we're using a live weather option.
> +	function getMusic() {
> +		if(isLive()) {
> +			if(weatherManager.getWeather() == "Rain")
> +				return "new-leaf-raining";

The extension doesn't have any audio files for WW/CF raining

—Reply to this email directly or view it on GitHub.